mapbox/dr-ui
# add `isStuck` prop to `TopBarSticker` component

I would like to propose adding an `isStuck` prop to `TopBarSticker`. This prop's default is `true and allows the option to set it to `false` for occasions where we do not want a sticky top bar.

I'm keen on making the TopBarSticker static on /playground since there aren't any navigational items in the top bar and it takes up valuable screen space. Ref https://github.com/mapbox/playground/issues/78

🐌 


## How to test

1. Run `npm start`
2. open /TopbarSticker test case
3. The last test case should _not_ stick.

## QA checklist




Open the test cases app locally on:
